UPGRADING

```diff
@@ -105,6 +105,10 @@ PHP 8.5 UPGRADE NOTES
     readline_callback_handler_install() have been changed to true, rather
     than bool.
 
+- opcache:
```

Comment:
```suggestion
- Opcache:
```

And also because the UPGRADING document is used as a starting point for the documentation, it should state that this is not portable.

Also a great weekend for you!